samples/subsys/fs/ext2/prj.conf

```diff
@@ -0,0 +1,11 @@
+CONFIG_LOG=y
+CONFIG_LOG_MODE_IMMEDIATE=y
+
+CONFIG_FILE_SYSTEM=y
+CONFIG_FILE_SYSTEM_EXT2=y
+
+CONFIG_DISK_ACCESS=y
+CONFIG_DISK_DRIVER_SDMMC=y
+
+# First block of first partition after GPT
+CONFIG_EXT2_DISK_STARTING_SECTOR=34
```

Comment:
Adding `CONFIG_MAIN_STACK_SIZE=2048` fixes the issue and is a minor change

Reply:
I have added it.